Fix #131 Add starter version of the Contributor Introduction Script (#136)
InnerSourceCommons/InnerSourceLearningPath: contributor/01-introduction-script.md

X: The Contributor role describes a person that makes contributions to the repos of an InnerSource community project.
This person may or may not be part or see themselves as part of the community although this often changes with continuing involvement with the community.

### Relationship to other roles

-> Possibly slide with the other two role names?

X: As a Contributor in an InnerSource community you will interact with people playing other roles of InnerSource, such as Trusted Committer or Product Owner and possibly also other contributors.

-> Flip open the TC role name on the slide

X: A Trusted Committer will be your host for your stay in the hosting community.
They are the gatekeepers to the project's code repository and will move your contribution closer to production once they accepted it. 
It is their role to mentor you on your way to contributing to their community, e.g. by providing good information to enable you to help yourself or help you in other cases.

-> Flip open the PO role name on the slide

X: Your interaction with the Product Owner will likely focus on working out the fit of your contribution into the general product and its road map. 

> You might work this role to make sure that general aspects of documentation or consistency of UI/UX are kept upon merging your contribution.  

Reviewer: ```suggestion
You might work with this role to make sure that general aspects of documentation or consistency of UI/UX are kept upon merging your contribution.  
```

However, there are some differences - depending on the size of the project, this role is often filled by the same person that acts as Trusted Committer.

X: In many projects you will interact with other Contributors, too.

-> Flip open the Contributor name on the slide

X: We would like to encourage you to watch the video introducing the Trusted Committer role and the video on lowering the barriers to entry before delving deeper into the details of the Contributor role in this section.

-> Possibly hinting to some links popping up with the video links ~ YouTube style or just a short link overlay?

### Section overview

-> Possbly slide with the segment names, popping up one after another?

[EDITOR NOTE J:
I suggest alternating between segments.
Also, possibly shorten the texts.
]

> X: In the following 5 videos you will learn more in detail about the various aspects introduced before. 

Reviewer: ```suggestion
X: In the following 5 videos you will learn more details about the various aspects introduced before. 
```

The second segment, i.e. the one after this segment, will detail _mindset and habit_ that create opportunities to _become an InnerSource Contributor_.

X: In the third segment we'll look into the _ethos of the Contributor_ - i.e. aspects of _behavior_ that will lead to a pleasant and productive time for you and the host team and might spark more collaboration.
The guest-in-home analogy presented in the introductory videos will serve as a vivid example. 

X: The fourth segment describes the hands-on things to do to make your contribution a success - the _mechanics of Contributing_.
We'll give practical tips to leverage when preparing to work on a contribution, during development, and also in the pull request.  

> X: After we've dealt with the personal interaction-focused and technical of the contributor role, the fifth segment presents the benefits of making the effort to contribute from various perspectives: Your perspective, your team's perspective, and the company's perspective at large.

Reviewer: ```suggestion
X: After we've dealt with the personal, the interaction-focused, and the technical aspects of the contributor role, the fifth segment presents the benefits of making the effort to contribute from various perspectives: Your perspective, your team's perspective, and the perspective of the company at large.
```

> X: The last segment will recap what we've learned about being an InnerSource contributor.  We'll share how you can continue your learning on InnerSource both with with other online videos and also involvement with the online InnerSource community.

Reviewer: ```suggestion
X: The last segment will recap what we've learned about being an InnerSource contributor.  We'll share how you can continue your learning on InnerSource both with other online videos and involvement with the online InnerSource community.
```
